app/controllers/restful_redirect_controller.rb

```diff
@@ -8,8 +8,10 @@ def index
       if record
         if record.kind_of?(EmsConfiguration)
           redirect_to(:controller => 'ems_configuration', :action => 'show', :id => params[:id])
-        elsif %w[ManageIQ::Providers::AnsibleTower::AutomationManager].include?(record.type)
+        elsif %w[ManageIQ::Providers::ExternalAutomationManager].include?(record.type)
```

Comment:
I can't see how this would ever be true

Reply:
probably better for this to check  this?  (also the same for the embedded ansible check right after wards)
```suggestion
        elsif record.kind_of?(ManageIQ::Providers::ExternalAutomationManager)
```